docs/schedule/examples/flex.md

```diff
@@ -0,0 +1,326 @@
+
+[**booking_rules.txt**](../../reference/#booking_rulestxt)
+
+booking_rule_id | booking_type | prior_notice_start_day | prior_notice_start_time | prior_notice_last_day | prior_notice_last_time | message | phone_number | info_url
+-- | -- | -- | -- | -- | -- | -- | -- | --
+booking_route_74362 | 2 | 14 | 8:00:00 | 1 | 15:00:00 | Brown County Heartland Express provides door-to-door on-demand transportation. To request a ride, call 1-507-359-2717 or 1-800-707-2717 by 3pm at least one business day ahead of your trip. | (507) 359-2717 | https://www.co.brown.mn.us/heartland-express-transit
+
+### Define stop_times
+
+- The operating hours are defined using the `start_pickup_drop_off_window` and `end_pickup_drop_off_window` fields.
+- Travel within the same zone requires two records in stop_times.txt with the same `location_id`.
+  - The first record with `pickup_type = 2` and `drop_off_type = 1` indicates that booking boarding is allowed in the zone.
+  - The second record with `pickup_type = 1` and `drop_off_type = 2` indicates that booking alighting is allowed in the zone.
+ 
+[**stop_times.txt**](../../reference/#stop_timestxt)
+
+trip_id | location_id | stop_sequence | start_pickup_drop_off_window | end_pickup_drop_off_window | pickup_type | drop_off_type | pickup_booking_rule_id | drop_off_booking_rule_id
+-- | -- | -- | -- | -- | -- | -- | -- | --
+t_5374944_b_77497_tn_0 | area_715 | 1 | 06:15:00 | 08:00:00 | 2 | 1 | booking_route_74362 | booking_route_74362
+t_5374944_b_77497_tn_0 | area_715 | 2 | 06:15:00 | 08:00:00 | 1 | 2 | booking_route_74362 | booking_route_74362
+t_5374945_b_77497_tn_0 | area_708 | 1 | 08:00:00 | 17:00:00 | 2 | 1 | booking_route_74362 | booking_route_74362
+t_5374945_b_77497_tn_0 | area_708 | 2 | 08:00:00 | 17:00:00 | 1 | 2 | booking_route_74362 | booking_route_74362
+t_5374946_b_77497_tn_0 | area_715 | 1 | 17:00:00 | 17:45:00 | 2 | 1 | booking_route_74362 | booking_route_74362
+t_5374946_b_77497_tn_0 | area_715 | 2 | 17:00:00 | 17:45:00 | 1 | 2 | booking_route_74362 | booking_route_74362
+t_5374947_b_77497_tn_0 | area_715 | 1 | 08:00:00 | 12:00:00 | 2 | 1 | booking_route_74362 | booking_route_74362
+t_5374947_b_77497_tn_0 | area_715 | 2 | 08:00:00 | 12:45:00 | 1 | 2 | booking_route_74362 | booking_route_74362
+
+`area_715` refers to New Ulm City zone, `area_708` refers to Bronw County zone. 
+
+## On-demand services between multiple zones
+
+Some demand-responsive services operate between multiple distinct zones, where riders can book pickups at any point A within one area and drop-offs at any point B within another area. For example, [Minnesota River Valley Transit](https://www.saintpetermn.gov/330/Dial-a-Ride) offers on-demand services between Saint Peter and Kasota cities:
```

Comment:
A suggestion to make this paragraph clearer:

Some demand-responsive services operate **across** multiple zones, where riders can book pickups at any **location** within one area and drop-offs at any **location** within another area.

Also, here we use pickup/drop-off, and later in the page, we talk about using board/alight. 
I understand pickup/dropoff references the driver's perspective and board/alight references the rider's perspective, but if there is a way to only use one, it would increase clarity.

Reply:
Updated text, rest of document has been harmonized to use pickup/dropoff